README.md

```diff
@@ -27,7 +27,7 @@ Currently, `client-go` is in version `v4.0.0-beta.0`.
 
 ## Container Image
 
-The latest container image can be found at `gcr.io/google_containers/kube-state-metrics:v0.5.0`.
+The latest container image can be found at `quay.io/coreos/kube-state-metrics:v1.0.1`.
```

Comment:
Let's enumerate both container image repositories here, but write a note, that the gcr image is only maintained on best effort as it requires external action from Google employees as no Google employee actively works on kube-state-metrics.

Reply:
done. PTAL.